modules/core/src/v2/coins/xlm.ts

```diff
@@ -759,4 +769,125 @@ export class Xlm extends BaseCoin {
       return true;
     }).call(this).asCallback(callback);
   }
+
+  /**
+   * Derive a hardened child public key from a master key seed using an additional seed for randomness.
+   *
+   * Due to technical differences between keypairs on the ed25519 curve and the secp256k1 curve,
+   * only hardened private key derivation is supported.
+   *
+   * @param key seed for the master key. Note: Not the public key or encoded private key. This is the raw seed.
+   * @param entropySeed random seed which is hashed to generate the derivation path
+   */
+  deriveKeyWithSeed({ key, seed }: { key: string; seed: string }): { derivationPath: string; key: any } {
+    const derivationPathInput = bitcoin.crypto.hash256(`${seed}`).toString('hex');
+    const derivationPathParts = [
+      999999,
```

Comment:
What does this magic number represent?

Reply:
No clue, I copied it from the baseCoin implementation.

https://github.com/BitGo/BitGoJS/blob/75977994b406ace9c69634a6e43a6e5d06dac3c7/modules/core/src/v2/baseCoin.ts#L248

Reply:
I believe this number is fixed so we group all our derived keys under the same root tree: https://github.com/bitcoin/bips/blob/master/bip-0032/derivation.png

But I'm not sure of the benefits.